### PR TITLE
Fix some errors and add tests

### PR DESCRIPTION
**Pull request by sujeebant**
indexedDBUtils.js: fix indexedDB calls
test/masq.test.js: fix calls to masq.init
test: add indexedDBUtils test
package-lock.json: update file

### CODE REVIEW

File: test/indexedDBUtils.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const indexedDBUtils = require('../src/indexedDBUtils')
+
+window.indexedDB = require('fake-indexeddb')
+
+test('DB should exist', async () => {
+  expect.assertions(1)
```
Comment (G-Ray) [inline test/indexedDBUtils.test.js]: You should not need to expect.assertion, this is required for asynchronous tests only.

File: test/indexedDBUtils.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  window.indexedDB.open('test1')
+  expect(await indexedDBUtils.dbExists('test1')).toBe(true)
+})
+
+test('DB should not exist', async () => {
+  expect.assertions(1)
+  window.indexedDB.open('test2')
```
Comment (G-Ray) [inline test/indexedDBUtils.test.js]: You shouldn't need to call `open('test2')` for this tests case right ?

File: package.json [hunk 3]
```diff
@@ -31,6 +31,7 @@
     "webpack-cli": "^3.1.2"
   },
   "dependencies": {
+    "fake-indexeddb": "^2.0.4",
```
Comment (G-Ray) [inline package.json]: Please move this to the `devDependencies` section, as it is not required during the lib execution.

File: test/indexedDBUtils.test.js (hunk 1 above)
Comment (sujeebant) [reply test/indexedDBUtils.test.js]: done

File: package.json (hunk 3 above)
Comment (sujeebant) [reply package.json]: done

File: test/indexedDBUtils.test.js (hunk 2 above)
Comment (sujeebant) [reply test/indexedDBUtils.test.js]: done

**Pull request merged**